src/backend/distributed/shared_library_init.c

```diff
@@ -448,16 +448,16 @@ RegisterCitusConfigVariables(void)
 		NULL, NULL, NULL);
 
 	DefineCustomEnumVariable(
-		"citus.copy_transaction_manager",
-		gettext_noop("Sets the transaction manager for COPY into distributed tables."),
-		gettext_noop("When a failure occurs during when copying into a distributed "
-					 "table, 2PC is required to ensure data is never lost. Change "
+		"citus.multi_shard_commit_protocol",
+		gettext_noop("Sets the commit protocol for distributed transactions."),
+		gettext_noop("When a failure occurs during an operation inside a distributed "
```

Comment:
Instead of "distributed transactions", maybe "commands that modify multiple shards". We might also want to add that this currently only applies to COPY.